internal/resources/iampolicy/role_binding_test.go

```diff
@@ -65,7 +65,7 @@ func TestRoleBindingListToUpdate(t *testing.T) {
 		{
 			description: "check for normal scenario of an entry in map",
 			input: &map[string]iammodel.VmwareTanzuCoreV1alpha1PolicyBindingDeltaOpType{
-				"cluster.admin_test_USER": iammodel.VmwareTanzuCoreV1alpha1PolicyBindingDeltaOpTypeADD,
+				"cluster.admin,test,USER": iammodel.VmwareTanzuCoreV1alpha1PolicyBindingDeltaOpTypeADD,
```

Comment:
Can we use semicolon or - or dot instead.

Reply:
Please use semicolon or -

Reply:
switched to semicolon